### ISSUE TITLE
Prevent Wiki from being editable by anybody

### ISSUE THREAD

**Issue by p-fischer** [square/retrofit#3776]
https://github.com/square/retrofit/wiki currently seems to be editable by anybody and this seems to be exploited.

**Comment by kelunik** [comment]
Indeed, e.g. I've removed a probably malicious download link here: https://github.com/square/retrofit/wiki/Home/_compare/4a0aca53c58e56018bd2f0c5607f74878affee13...8d5a4affcfd150fa06ba9799fb18b1a78cc8f580

**Comment by JakeWharton** [comment]
There's no need to have a wiki if it's not publicly editable. We can move the content to the website and force people to do PRs instead.

**Comment by JakeWharton** [comment]
The wiki is gone. Moved to website.
